src/exercise/index.cjsx

```diff
@@ -34,6 +36,14 @@ ExerciseBase = React.createClass
   contextTypes:
     processHtmlAndMath: React.PropTypes.func
 
+  renderHelpLink: (sections) ->
+    if not sections?.length then return
+
+    section = _.first(sections)
+    <div key='task-help-links' className='task-help-links'>
+      Comes from {@sectionFormat(section.chapter_section)} - {section.title}
```

Comment:
I wonder how often we have more than one relevant related content -- probably something we can look at later